postgres/functions/monitoring_ingest_couchpg_seqs.sql

```diff
@@ -25,12 +25,14 @@ BEGIN
                 credentials.user,
                 credentials.password
               ), 
-'select
-  current_database() as partner_name,
-  date_trunc(''day'', now()) as created,
-  split_part(seq, ''-'', 1) as seq,
-  split_part(source, ''/'', 2) as source
-from couchdb_progress;',
+'SELECT
+  current_database() AS partner_name,
+  date_trunc(''day'', now()) AS created,
+  split_part(seq, ''-'', 1) AS seq,
+  split_part(source, ''/'', 2) AS source
+FROM couchdb_progress
+WHERE seq != ''0''
```

Comment:
I believe '0' might be valid for cases when couch2pg errors out from the time it was set up.

Reply:
totally. do you think this is an inappropriate fix? my assumption here is we are mostly monitoring systems that have worked at least once - rather than using this as diagnostic tool for first-time use?  unsure.

open to suggestions!